### PR TITLE
Fix to issue 23326 for completions on new.target

### LINKED ISSUE #23326: No completions on meta-properties
```ts
class C {
  constructor() {
    new./**/
  }
}
```

**Expected**: Completion for the `target` keyword at the `/**/` marker.
**Actual**: No completions.

### PR DESCRIPTION
**Pull request by elizabethdinella**
Fixes #23326

### CODE REVIEW

File: src/services/completions.ts [hunk 1]
```diff
@@ -888,6 +888,9 @@ namespace ts.Completions {
                     case SyntaxKind.ImportType:
                         node = parent;
                         break;
+                    case SyntaxKind.MetaProperty:
```
Comment (RyanCavanaugh) [inline src/services/completions.ts]: Move two lines up and delete duplicate code

File: tests/cases/fourslash/completionsNewTarget.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+////class C {
+////    constructor() {
+////        if (C === new./**/)
+////    }
+////}
+
+goTo.marker("");
+debugger;
```
Comment (RyanCavanaugh) [inline tests/cases/fourslash/completionsNewTarget.ts]: Remove debugger statement

File: src/services/completions.ts [hunk 3]
```diff
@@ -1050,6 +1051,11 @@ namespace ts.Completions {
                 }
             }
 
+            if (isMetaProperty(node) && node.keywordToken === SyntaxKind.NewKeyword) {
```
Comment (mhegazy) [inline src/services/completions.ts]: `import.meta` is another meta property that we should support.

**Comment by RyanCavanaugh** [comment]
(thought I had a suggestion but I didn't)

**Pull request merged**